src/main.cpp

```diff
@@ -81,6 +82,7 @@ static void help(FILE *fp)
         "                               <error|warning|info|debug>\n"
         "  -v --verbose                 Verbose. Same as --debug-log-level=debug\n"
         "  -V --version                 Show version\n"
+        "  -s --sniffer-sysid           Sysid that all messages are sent to. 0 to disable\n"
```

Comment:
"0 to disable" doesn't match your check in line 246 ff.

Reply:
Added range check and fixed message.